moped-editor/src/utils/activityLogFormatters/mopedProjectTypesActivity.js

```diff
@@ -0,0 +1,38 @@
+import BeenhereOutlinedIcon from "@material-ui/icons/BeenhereOutlined";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatProjectTypesActivity = (change, projectTypeList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_project_types"];
```

Comment:
Ran this locally and noticed there is a compile warning that this var is never used. Might be used down the line but wanted to note that